tf_image_segmentation/recipes/pascal_voc/FCNs/fcn_32s_train.ipynb

```diff
@@ -2,11 +2,19 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
    "metadata": {
-    "collapsed": true
+    "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['', '/home/vittal/anaconda2/lib/python2.7/site-packages/progressbar2-3.6.2-py2.7.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages/picklable_itertools-0.1.1-py2.7.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages/argparse-1.4.0-py2.7.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages/fuel-0.2.0-py2.7-linux-x86_64.egg', '/home/vittal/anaconda2/lib/python27.zip', '/home/vittal/anaconda2/lib/python2.7', '/home/vittal/anaconda2/lib/python2.7/plat-linux2', '/home/vittal/anaconda2/lib/python2.7/lib-tk', '/home/vittal/anaconda2/lib/python2.7/lib-old', '/home/vittal/anaconda2/lib/python2.7/lib-dynload', '/home/vittal/anaconda2/lib/python2.7/site-packages/Sphinx-1.3.5-py2.7.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages/setuptools-23.0.0-py2.7.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages', '/home/vittal/anaconda2/lib/python2.7/site-packages/cryptography-1.0.2-py2.7-linux-x86_64.egg', '/home/vittal/anaconda2/lib/python2.7/site-packages/IPython/extensions', '/home/vittal/.ipython', '/home/dpakhom1/workspace/my_models/slim/', '/home/dpakhom1/tf_projects/segmentation/tf-image-segmentation/']\n"
```

Comment:
The personal paths have been removed. What you have pointed out to is a stray output as a result of me testing something. I will clean it up. But, you should be able to add a block in set_paths.py pointing to your system specific paths and you should be able to get it to run.